src/catalog/catalog_manager.py

```diff
@@ -256,6 +256,40 @@ def create_udf(self, name: str, impl_file_path: str,
         self._udf_io_service.add_udf_io(udf_io_list)
         return metadata
```

Comment:
We could add a README in this directory listing the schemata of all the catalog tables.

Reply:
Yeah, even I feel the need. I will add this to the ToDos. I will skip this now as we need this PR to enable integration test cases.